Exiled.API/Features/Player.cs

```diff
@@ -1247,14 +1282,70 @@ public void EnableEffect<T>(float duration = 0f, bool addDurationIfActive = fals
             where T : PlayerEffect => ReferenceHub.playerEffectsController.EnableEffect<T>(duration, addDurationIfActive);
 
         /// <summary>
-        /// Enables a status effect on this player.
+        /// Enables a <see cref="PlayerEffect">status effect</see> on the player.
         /// </summary>
         /// <param name="effect">The name of the <see cref="PlayerEffect"/> to enable.</param>
         /// <param name="duration">The amount of time the effect will be active for.</param>
         /// <param name="addDurationIfActive">If the effect is already active, setting to true will add this duration onto the effect.</param>
         /// <returns>A bool indicating whether or not the effect was valid and successfully enabled.</returns>
         public bool EnableEffect(string effect, float duration = 0f, bool addDurationIfActive = false) => ReferenceHub.playerEffectsController.EnableByString(effect, duration, addDurationIfActive);
 
+        /// <summary>
+        /// Enables an array of <see cref="PlayerEffect">status effects</see>.
+        /// </summary>
+        /// <param name="effects">An array of <see cref="PlayerEffect"/>s by name.</param>
+        /// <param name="duration">The duration to apply to all of the effects.</param>
+        /// <param name="addDurationIfActive">If the effect is already active, setting to true will add this duration onto the effect.</param>
+        /// <returns>A <see cref="bool"/> indicating whether or not all of the effects were applied successfully.</returns>
+        public bool EnableEffects(string[] effects, float duration = 0f, bool addDurationIfActive = false)
+        {
+            bool flag = true;
+            foreach (string str in effects)
+            {
+                bool success = ReferenceHub.playerEffectsController.EnableByString(str, duration, addDurationIfActive);
+                if (!success)
+                {
+                    flag = false;
+                }
+            }
+
+            return flag;
+        }
```

Comment:
Same as with `GetActiveEffects()`, does anyone need to enable a bunch of events with the same options?

Reply:
I've had a case before where I could've used this but it wasn't available, although I see your point

Reply:
I mean, it'd be very useful.. but using types instead of string